addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -196,15 +197,29 @@
             }).then(function (quiz_data) {
                 self.slide.sessionAnswers = quiz_data.session_answers;
                 self.quiz = {
+                    description: quiz_data.slide_description,
                     questions: quiz_data.slide_questions || [],
                     questionsCount: quiz_data.slide_questions.length,
                     quizAttemptsCount: quiz_data.quiz_attempts_count || 0,
                     quizKarmaGain: quiz_data.quiz_karma_gain || 0,
                     quizKarmaWon: quiz_data.quiz_karma_won || 0,
+                    slideResources: quiz_data.slide_resource_ids || [],
                 };
             });
         },
 
+        /*
+        * Render description (Markup).
+        *
+        * @private
+        */
+        _renderDescription: function () {
+            const $descriptionElem = document.getElementsByClassName('o_wslides_js_lesson_quiz_description');
+            if($descriptionElem[0]){
+                $descriptionElem[0].innerHTML = Markup(this.quiz.description || '');
```

Comment:
Isn't it possible to do ``<t t-out="widget.quiz.description_safe"/>`` where description safe is the markup of the received description, stored on this.quiz ?

Reply:
thanks a lot for checking everything! t-out indeed seems to respect markupized content, so I changed it accordingly.